assets/src/edit-story/elements/video/display.js

```diff
@@ -62,19 +64,31 @@ function VideoDisplay({
   );
   return (
     <Element>
-      <Video
-        poster={poster || resource.poster}
-        style={style}
-        {...videoProps}
-        loop={loop}
-      >
-        <source src={resource.src} type={resource.mimeType} />
-      </Video>
+      {previewMode ? (
+        <img
+          src={poster || resource.poster}
+          alt={resource.title}
+          width={videoProps.width}
```

Comment:
This will not do or probably not even enough. But the good news is that we should everything we need here. We just need a `PreviewImage` - a clone of `const Video = styled.video` above. And we should also pass `style={style}` and `{...videoProps}` like for video. Basically, all the layout nuances should be exactly the same. The only difference is that one is an img tag and the other is a video tag.